keyboards/skeletonkbd/frost68/info.json

```diff
@@ -0,0 +1,1235 @@
+{
+	"manufacturer": "skeletonkbd",
+	"keyboard_name": "Frost68",
+	"maintainer": "skeletonkbd",
+	"bootloader": "rp2040",
+	"diode_direction": "COL2ROW",
+	"features": {
+		"bootmagic": true,
+		"command": false,
+		"console": false,
+		"extrakey": true,
+		"mousekey": true,
+		"nkro": true,
+		"rgb_matrix": true
+	},
+	"matrix_pins": {
+		"cols": [
+			"GP21",
+			"GP22",
+			"GP23",
+			"GP24",
+			"GP25",
+			"GP26",
+			"GP28",
+			"GP3",
+			"GP4",
+			"GP5",
+			"GP6",
+			"GP9",
+			"GP10",
+			"GP11",
+			"GP12",
+			"GP13"
+		],
+		"rows": [
+			"GP29",
+			"GP27",
+			"GP20",
+			"GP19",
+			"GP18"
+		]
+	},
+	"processor": "RP2040",
+	"url": "https://skeletonkbd.com/products/frost68",
+	"usb": {
+		"device_version": "1.0.0",
+		"vid": "0x1209",
```

Comment:
pid.codes. For reference:
https://pid.codes/1209/68A0/

Reply:
Changed url. Thank you.
573c26589149f2f007e78c4f89cec8d58d00efd7

Reply:
Thats not what was asked of you, you need to change the value of `"vid"` to something else. The provided url was just to prove that the value was already in use.

Reply:
I submitted 1209/68A0 to pid.codes.
https://github.com/pidcodes/pidcodes.github.com/pull/818

Changed the `url` in `info.json` to the same as the one applied for in pid.codes.
cbcccc3c7824f336e2cbebb15cee92a9f8594a5e

Is there anything else I should do?